docs/source/concept_models/vivarium_gates_bep/concept_model.rst

```diff
@@ -170,8 +170,8 @@ Here is a memo describing the rationale underlying the causal structure of this
      - same 
    * - b (∆oMMN)
      - MMN vs IFA on birthweight in g
-     - +51.24 (43.71 to 58.76) 
-     - fixed effects meta analysis of 13 trials from Keats 2019 :download:`memo <meta-analysis_MMN_vs_IFA_memo.docx>`
+     - +51.24 (43.71 to 58.76) fixed effects *or* +45.16 (32.31 to 58.02) random effects
```

Comment:
We decided to use the RE version in the sim, right?  Good to have both here, but also let's document clearly which one we have decided to use.

Reply:
yes, I made a note in the bottom of the table to use random effect(s).